.github/workflows/ci-cd.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
       - name: Lint charts
-        run: ct lint --charts=charts/trino --validate-maintainers=false
+        run: ct lint --charts=charts/trino,charts/gateway --validate-maintainers=false
```

Comment:
Move this into a separate commit - the next one that also fixes the linting issues.